test/contracts/aevm_3/acm_dutch_auction.aes

```diff
@@ -0,0 +1,25 @@
+contract DutchAuction =
```

Comment:
Could you please point me to where this file is hooked to CI?

Reply:
`aehttp_contract_SUITE`

Reply:
https://github.com/aeternity/aeternity/pull/2398/commits/c2400163f4ce453bc47d687d4f7f5844e37065bc ?
Thanks.